support/scripts/build_spracingh7extreme.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/sh
```

Comment:
This shouldn't be here.

Reply:
This is normal for the headers of scripts. This identifies the interpreter.

Reply:
> This is normal for the headers of scripts. This identifies the interpreter.

I meant the whole file ;)

Reply:
Oh the entire file is not required? In that case, possibly yes as this detail belongs in CICD systems instead of source repos.

Reply:
yes, I mentioned that the commit that adds the file can be deleted or moved into an examples folder to serve as a reference for anyone else in the same position i was in when trying to work all this out.